### PR TITLE
added changelog/master.xml directory to resourcesec added the ability to grant users administrator status

### PR DESCRIPTION
**Pull request by sanv1ad**

### CODE REVIEW

File: src/main/java/cmps/charityservice/SampleController.java [hunk 1]
```diff
@@ -1,20 +1,36 @@
...
+import org.keycloak.admin.client.resource.UsersResource;
+import org.keycloak.representations.idm.RoleRepresentation;
+import org.keycloak.representations.idm.UserRepresentation;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.access.annotation.Secured;
 import org.springframework.security.oauth2.server.resource.authentication.JwtAuthenticationToken;
-import org.springframework.web.bind.annotation.CrossOrigin;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
```
Comment (MartianDill) [inline src/main/java/cmps/charityservice/SampleController.java:7]: You have removed these import lines 4-7, even though they are referenced in lines 11-13 and 16
Delete the lines that refer to them entirely if they are not needed or restore the deleted imports

File: src/main/resources/changelog/master.xml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+</databaseChangeLog>
```
Comment (MartianDill) [inline src/main/resources/changelog/master.xml:5]: In my branch I also created a similar file for myself, in my version it looked like this:
```java
<?xml version="1.0" encoding="UTF-8"?>
<databaseChangeLog
        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
    <includeAll path="initial_schema" relativeToChangelogFile="true"/>
</databaseChangeLog>
``` 
Maybe you should add lines with xml version and 'includeAll' too

**Review by MartianDill — CHANGES_REQUESTED** [review]
Requested change on 

- [ ] SampleController.java

- [x] master.xml

**Review by Misha999777 — APPROVED** [review]
Added some fixes, let's discuss them at the next meeting

**Pull request merged**